crates/protocol/src/cgo.rs

```diff
@@ -135,7 +135,7 @@ impl Trampoline {
             }
         });
 
-        tracing::debug!(?id, ?code, queued = ?queue.1.len(), "queued trampoline task");
+        tracing::debug!(?id, ?code, queued = ?queue.1.len(), awaiting = ?self.awaiting.borrow().len(), "queued trampoline task");
```

Comment:
Similar question to my last one: how big is the `awaiting` queue getting?

Reply:
I've seen it get into the 20-30s in normalish operation, and much larger (10's of thousands) when node was getting wedged.